src/utils/validations.ts

```diff
@@ -111,11 +111,11 @@ export function validateVaultWithdraw(props: ValidateVaultWithdrawProps): Valida
   if (yvTokenAmount.isZero()) return {};
 
   if (amountInWei.lt(0)) {
-    return { error: 'INVALID AMOUNT' };
+    return { error: 'Invalid amount' };
   }
 
   if (amountInWei.gt(userYvTokenBalance)) {
-    return { error: 'INSUFFICIENT FUNDS' };
+    return { error: 'Insufficient funds' };
```

Comment:
we should create another PR in the future and have error codes or something like "INSUFFICIENT_FUNDS" so we can just translate them and have them all in a .json file